.github/workflows/deploy-vue-storefront-cloud.yml

```
@@ -86,7 +87,7 @@ jobs:
             exit 1
           fi
       - name: Deploy on demo-magento2-canary.europe-west1.gcp.storefrontcloud.io
-        if: startsWith(github.ref, 'refs/heads/release')
+        if: startsWith(github.ref, 'refs/heads/refactor/perf-improvements')
```

```suggestion
        if: startsWith(github.ref, 'refs/heads/release')
```
I assume those are just leftovers from the refactor/perf-improvements branch? If it won't exist anymore, the changes in this .yml file aren't needed, no?